apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -0,0 +1,1124 @@
+                      }
+                    }}
+                    toggleVote={() => {
+                      if (isSelected) {
+                        if (isFocused) {
+                          setFocusedOptionId('');
+                        }
+                        updateVote(optionId, false);
+                        updateWriteInState(optionId, { type: 'invalid' });
+                      } else {
+                        updateVote(optionId, true);
+                        updateWriteInState(optionId, { type: 'pending' });
+                      }
+                    }}
+                    officialCandidateNames={officialCandidates
+                      .filter(
+                        (c) =>
+                          !selectedCandidateNames.includes(c.name) ||
+                          (isExistingCandidate(writeInEntry) &&
+                            writeInEntry.name === c.name)
+                      )
+                      .map((c) => c.name)}
+                    writeInCandidateNames={writeInCandidates
+                      .filter(
+                        (c) =>
+                          !selectedCandidateNames.includes(c.name) ||
+                          (isExistingCandidate(writeInEntry) &&
+                            writeInEntry.name === c.name)
+                      )
+                      .map((c) => c.name)}
+                    caption={renderCandidateButtonCaption({
+                      originalVote,
+                      currentVote: isSelected,
+                      isWriteIn: true,
+                      writeInRecord,
+                      writeInState: writeInEntry,
+                    })}
+                  />
+                );
+              })}
+            </CandidateButtonList>
+          )}
+          <BallotFooter>
+            <BallotMetadata>
+              <SmallText>
+                {scrollIndex + 1} of {numBallots}
+              </SmallText>
+              <SmallText>Ballot ID: {currentCvrId?.substring(0, 4)}</SmallText>
+            </BallotMetadata>
+            <BallotNavigation>
+              <Button
+                disabled={scrollIndex === 0}
+                icon="Previous"
+                onPress={() => {
+                  setScrollIndex(scrollIndex - 1);
+                  setIsStateStale(true);
+                }}
+                style={{ width: '5.5rem' }}
+              >
+                Back
+              </Button>
+              <Button
+                disabled={onLastBallot}
+                onPress={() => {
+                  setScrollIndex(scrollIndex + 1);
+                  setIsStateStale(true);
+                }}
+                rightIcon="Next"
+                style={{ width: '5.5rem' }}
+              >
+                Skip
+              </Button>
+              <Button
+                disabled={!allWriteInsAdjudicated}
+                icon="Done"
+                onPress={saveAndNext}
+                style={{ flexGrow: '1' }}
```

Comment:
We've got a bunch of `styled` components above. Add these too, e.g. `SecondaryNavButton` & `PrimaryNavButton` or `BackSkipButton` & `SaveButton`?